.travis.yml

```diff
@@ -11,6 +11,7 @@ cache:
 
 install:
```

Comment:
you could include an atmosphere/settings/local/travis.py file in the respository and copy in to the atmosphere/settings/local.py here.